menpo/image/base.py

```diff
@@ -2525,17 +2530,38 @@ def pixels_range(self):
 
     def rolled_channels(self):
         r"""
+        Deprecated - please use the equivalent ``as_rolled_channels`` method.
+        """
+        warn('This method is no longer supported and will be removed in a '
+             'future version of Menpo. '
+             'Use .as_rolled_channels() instead.',
+             MenpoDeprecationWarning)
+        return self.as_rolled_channels()
+
+    def as_rolled_channels(self, out_dtype=None):
```

Comment:
I see your point and if that is the consensus I'll change it to `.pixels_with_channels_at_back`.

However, if the point of the object is just to have a thin wrapper around a ndarray representing an image, a `.to_array(channels_at_back=False)` would be as coherent.

`image.to_array()` has the expected behaviour and `image.to_array(channels_at_back=True)` is very clear. The advantage being the possible addition of other optional parameters such as normalise etc. Also, in practice, relying solely on users tab-completing is not very robust and can result in an overly long list.  Again, just a suggestion.

In any case, I updated the PR to include the other comments so unless there are other comments, the it should be good to merge once that naming issue is solved :)